sopel/logger.py

```diff
@@ -106,6 +106,11 @@ def setup_logging(settings):
                 'propagate': False,
                 'handlers': ['exceptionfile'],
             },
+            # asyncio logging
+            'asyncio': {
+                'level': 'DEBUG',
+                'handlers': ['console'],
+            },
```

Comment:
Oh yeah, I forgot to check if it outputs anything if asyncio isn't used in debug mode (either by an env var or with using the debug parameter).

Reply:
So it's only useful in debug mode, if you have some env var setup to get debug information. It's really just for development purpose, nothing comes out otherwise, as far as I can tell.